ee-feature-pack/common/javadoc/extractPublicApiArtifactsList.sh

```diff
@@ -69,10 +69,10 @@ function convertModuleNameToGA() {
 convertModuleNameToGA $TARGET/modules.include.txt $TARGET/artifacts.in.tmp.txt;
 convertModuleNameToGA $TARGET/modules.exclude.txt $TARGET/artifacts.ex.tmp.txt;
 
-###  Blacklisted ->  prepend '#'.
+###  Blocklisted ->  prepend '#'.
   echo > $TARGET/artifacts.in.tmp-blist.txt;
   while read -r ARTIFACT ; do
-    if grep --quiet "$ARTIFACT\$" $DIRNAME/artifactsBlacklist.txt ; then echo -n "# " >> $TARGET/artifacts.in.tmp-blist.txt; fi
+    if grep --quiet "$ARTIFACT\$" $DIRNAME/artifactsBlocklist.txt ; then echo -n "# " >> $TARGET/artifacts.in.tmp-blist.txt; fi
```

Comment:
You'll need to rename this file too:

https://github.com/wildfly/wildfly/blob/main/ee-feature-pack/common/javadoc/artifactsBlacklist.txt

Reply:
PR Updated. Thanks